### PR TITLE
adding SDK guides for readme.io

### PR DESCRIPTION
**Pull request by fscelliott**
for readme-sync

### CODE REVIEW

File: docs/010 - install-sdk-go.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Install SDK"
+slug: "install-sdk-go"
+hidden: false
+createdAt: "2019-08-21T21:13:10.351Z"
```
Comment (mikeng13) [inline docs/010 - install-sdk-go.md:5]: Are these timestamps needed? If so, @juancarlostong during the syncing process, if there are changes, we should probably look at updating the timestamps for the `updatedAt` field

File: docs/020 - initialize-sdk-go.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+hidden: false
+createdAt: "2019-08-21T21:13:33.317Z"
+updatedAt: "2019-08-21T21:13:45.319Z"
+---
+To initialize the **OptimizelyClient** you will need either the SDK key or hard-coded JSON datafile.
+[block:code]
+{
+  "codes": [
+    {
+      "code": "import \"github.com/optimizely/go-sdk/optimizely/client\"\n\noptimizelyFactory := &client.OptimizelyFactory{\n          SDKKey: \"[SDK_KEY_HERE]\",\n          Datafile: []byte(\"DATAFILE_JSON_STRING_HERE\")\n}\n\n// Instantiates a static client (no datafile polling)\noptlyClient, err := optimizelyFactory.StaticClient()\n\n// Instantiates a client that syncs the datafile in the background\noptlyClient, err := optimizelyFactory.Client()\n\n",
```
Comment (mikeng13) [inline docs/020 - initialize-sdk-go.md:13]: Heh this is super hard to read on Github. Do you use a formatter locally to verify the structure and visual aspect of this code block? Was wondering if we can instead use GH markdown or some more commonly accepted MD format

File: docs/010 - install-sdk-go.md (hunk 1 above)
Comment (mikeng13) [reply docs/010 - install-sdk-go.md:5]: No need to change the contents here. Merely wanna make sure that we take these into account.

File: docs/020 - initialize-sdk-go.md (hunk 2 above)
Comment (fscelliott) [reply docs/020 - initialize-sdk-go.md:13]: @mikeng13 yes, I think we talked about this earlier; I recommended that you get someone (I think you mentioned farming out to a contractor, maybe?) to migrate these docs from ReadMe MD to GH markdown, for all languages.

File: docs/010 - install-sdk-go.md (hunk 1 above)
Comment (fscelliott) [reply docs/010 - install-sdk-go.md:5]: @mikeng13  yeah I think the timestamp is important, I'm guess it's what powers the 'last updated' line: 
image

**Pull request merged**